packages/core/content-manager/admin/src/utils/api.ts

```diff
@@ -32,12 +32,6 @@ const buildValidParams = <TQuery extends Query>(query: TQuery): TransformedQuery
     ),
   };
 
-  if ('_q' in validQueryParams) {
-    // Encode the search query here since the paramsSerializer will not
-    // @ts-expect-error – TODO: fix this type error
-    validQueryParams._q = encodeURIComponent(validQueryParams._q);
-  }
```

Comment:
Isn't buildValidParams used in a lot of places? Did you check the impact removing this has?

Reply:
not yet! I will do the QA today